packages/libs/react-ui/src/components/Accordion/Accordion.css.ts

```diff
@@ -37,18 +37,36 @@ export const accordionButtonClass = style([
 
 export const accordionContentClass = style([
   atoms({
-    display: 'none',
+    display: 'grid',
     color: 'text.subtle.default',
     fontSize: 'base',
     margin: 'no',
     overflow: 'hidden',
+    paddingInlineStart: 'sm',
+    height: '100%',
+    paddingBlock: 'no',
+  }),
+  {
+    gridTemplateRows: '0fr',
+    transition: 'all 300ms ease',
+  },
+]);
+
+export const accordionContentOpenClass = style([
+  atoms({
     paddingBlock: 'sm',
   }),
   {
-    selectors: {
-      "&[data-open='true']": {
-        display: 'block',
-      },
-    },
+    gridTemplateRows: '1fr',
   },
 ]);
+
+export const accordionContentWrapperClass = style([{ minHeight: '0' }]);
```

Comment:
please ensure that the tools app is working fine, I had the animation before and reverted it because there was an issue with tools and didn't had time to fix back then

Reply:
I'll test it to be sure, do you remember what the issue was?

Reply:
Checked and works like a charm!